ligo/stablecoin/actions.ligo

```diff
@@ -52,7 +142,96 @@ function debit_from
       }
   | None -> failwith ("Given address is not present in ledger")
   end
-} with store with record [ ledger = updated_ledger ]
+} with store with record
+    [ ledger = updated_ledger
+    ; total_supply = abs (store.total_supply - parameter.amount) // TODO: possible bug here
+    ]
+
+(*
+ * Converts a transfer item into a safelist transfer item
+ *)
+function convert_to_safelist_transfer
+  (const tp : transfer_param
+  ) : safelist_transfer_item is block
+  { const sl_transfer_item : safelist_transfer_item =
+      (record
+        [ from_ = tp.0
+        ; to_ = tp.1.0
+        ] : safelist_transfer_item)
+
+  } with (sl_transfer_item)
+
+(*
+ * Calls `assert_transfer` of the provided safelist contract using
+ * the provided transfer_descriptors.
+ *)
+function call_assert_transfers
+  ( const ops_in : list(operation)
```

Comment:
I guess this argument can be omitted since you always call it with `nil`

Reply:
Changed here and below.